### PR TITLE
[FB] PMM-5813 fix broken wizard

### PR DESCRIPTION
**Pull request by Sevlin**
Use wizard page (password-page) for different installation types (OVF, AMI, Docker)
---
Related PRs:
- [pmm-submodules](https://github.com/Percona-Lab/pmm-submodules/pull/930) - fix permissions
- [pmm-submodules](https://github.com/Percona-Lab/pmm-submodules/pull/933) - modified Dockerfiles
- [pmm-server](https://github.com/percona/pmm-server/pull/213)
- [pmm-server-packaging](https://github.com/percona/pmm-server-packaging/pull/138)
- [percona-images](https://github.com/Percona-Lab/percona-images/pull/75)

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-a042612
client docker - perconalab/pmm-client-fb:PR-885-a042612
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-a042612.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-ece08e1
client docker - perconalab/pmm-client-fb:PR-885-ece08e1
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-ece08e1.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-8ae8208
client docker - perconalab/pmm-client-fb:PR-885-8ae8208
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-8ae8208.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-94ed063
client docker - perconalab/pmm-client-fb:PR-885-94ed063
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-94ed063.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-8e1fbf2
client docker - perconalab/pmm-client-fb:PR-885-8e1fbf2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-8e1fbf2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-055084b
client docker - perconalab/pmm-client-fb:PR-885-055084b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-055084b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-407402e
client docker - perconalab/pmm-client-fb:PR-885-407402e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-407402e.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-7712aa3
client docker - perconalab/pmm-client-fb:PR-885-7712aa3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-7712aa3.tar.gz

File: .gitmodules [hunk 1]
```diff
@@ -70,3 +70,7 @@
 	path = sources/pid-watchdog/src/github.com/percona/pid-watchdog
 	url = https://github.com/percona/pid-watchdog
 	branch = master
+[submodule "sources/percona-images"]
+	path = sources/percona-images
+    url = https://github.com/Percona-Lab/percona-images
+    branch = PMM-5813_password-page_per-dist
```
Comment (AlekSi) [inline .gitmodules:76]: It will not be used even if it's there. Please remove it

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-c40a6a8
client docker - perconalab/pmm-client-fb:PR-885-c40a6a8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-c40a6a8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-b8c81c6
client docker - perconalab/pmm-client-fb:PR-885-b8c81c6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-b8c81c6.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-b8c81c6
client docker - perconalab/pmm-client-fb:PR-885-b8c81c6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-b8c81c6.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-885-b8c81c6
client docker - perconalab/pmm-client-fb:PR-885-b8c81c6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm-client/pmm-client-PR-885-b8c81c6.tar.gz

**Pull request closed without merging**